e2e/rpc/rpc_test.go

```diff
@@ -138,6 +138,9 @@ func (s *RPCTestSuite) TestCallRawResult() {
 // TestCallContextResult checks if result passed to CallContext
 // is set accordingly to its underlying memory layout.
 func (s *RPCTestSuite) TestCallContextResult() {
+	// FIXME(tiabc): Stop skipping after https://github.com/status-im/status-go/issues/424
```

Comment:
This test should be run without the upstream. The upstream was used just to avoid waiting for the blockchain to sync.

Reply:
Thanks! Fixed.